tests/functional/object/mcg/test_noobaa_bucket_error_state_mode.py

```
@@ -0,0 +1,462 @@
+        bucket_name = create_unique_resource_name(
+            resource_description="bucket", resource_type="quota"
+        )
+        bucket = MCGCLIBucket(bucket_name, mcg=mcg_obj, quota="2Gi")
+        logger.info(f"Created bucket {bucket_name} with 2Gi quota")
+
+        try:
+            try:
+                write_random_test_objects_to_bucket(
+                    io_pod=awscli_pod,
+                    bucket_to_write=bucket_name,
+                    file_dir=f"/tmp/{bucket_name}",
```

Ruff (0.15.17)

[error] Probable insecure usage of temporary file or directory: "/tmp/"

(S108)